client/test-data/corsica.locoflex.rococo.json

```diff
@@ -0,0 +1,120 @@
+{
+  "type": "FeatureCollection",
+  "community": {
+    "meta": {
+      "name": "Corsica LoCoFlex",
+      "symbol": "CLL",
+      "assets": "QmP2fzfikh7VqTu8pvzd2G2vAd4eK7EaazXTEgqGN6AWoD",
+      "announcementSigner": {
+        "bip340": "0xceda72c19f11aebecf8fb4a0ab3fb2a03e03f283225655baab4728303a560a13"
```

Comment:
It doesn't make much sense to add an announcement signer which isn't a BIP340 / nostr account, because we'llnever be able to sign anything with it. But won't hurt because we can change metadata anytime

Reply:
Yeah, but I did not want to put `null` here, as I want to be able to test if the app understands the metadata if the value is set.